dev-infrastructure/templates/svc-infra-lookup.bicep

```diff
@@ -0,0 +1,9 @@
+@description('The name of the service keyvault')
+param serviceKeyVaultName string
+
+resource serviceKeyVault 'Microsoft.KeyVault/vaults@2024-04-01-preview' existing = {
+  name: serviceKeyVaultName
```

Comment:
there is a config override for the resourcegroup the svc keyvault lives in

https://github.com/Azure/ARO-HCP/blob/ab4df6c10a830a8c9f4f0bf8afe8094f693a6170/dev-infrastructure/configurations/svc-cluster.tmpl.bicepparam#L91

https://github.com/Azure/ARO-HCP/blob/1bc7db417f314c36f428f70ee7b4633891f182ac/dev-infrastructure/templates/svc-infra.bicep#L59

reason: in DEV the service KV is shared